### PR TITLE
[ADDED] Navbar

### PR DESCRIPTION
**Pull request by atulyadav745**
Responsive Navbar is added
Fixed issue #1

### CODE REVIEW

File: components/Navbar/NavLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+const NavLinks = () => {
+	const links = [{ name: 'Departments' }, { name: 'About Us' }]
+	return (
+		<>
+			{links.map((Link,i) => (
```
Comment (king-11) [inline components/Navbar/NavLinks.tsx]: Use a ul tag and flex box

File: components/Navbar/NavLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+				<div key={i}>
+					<div   className='px-3 text-left md:cursor-pointer' >
+						<h1 className='py-7'>{Link.name}</h1>
```
Comment (king-11) [inline components/Navbar/NavLinks.tsx]: Don't use h1 prefer span or p

File: components/Navbar/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Login = () => {
+  return (
+    <div>
```
Comment (king-11) [inline components/Navbar/Login.tsx]: Don't make this a component directly embed a button in navbar.tsx without div wrapper

File: components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            <IonIcon name={`${open ? "close" : "menu"}`}></IonIcon>
+          </div>
+        </div>
+        <ul className="md:flex hidden uppercase items-center gap-8 font-[Poppins]">
+          <li>
+            <Link href="/" className="py-7 px-3 inline-block">
+              Home
+            </Link>
+          </li>
+          <NavLinks />
```
Comment (king-11) [inline components/Navbar/Navbar.tsx]: Embed links here directly instead of a separate component

**Review by king-11 — CHANGES_REQUESTED** [review]
Combine into a single component and use semantic html instead of just div

**Comment by atulyadav745** [comment]
@king-11 Thanks Bhaiya. Great Review, hope I fixed it.
Screenshot (274)

Screenshot (275)
ng)
Screenshot (276)

**Comment by king-11** [comment]
@Divyansh013 we can use the new layout feature maybe configure that

**Pull request merged**